src/features/offer/pages/Offer.tsx

```diff
@@ -29,12 +31,20 @@ export const Offer: FunctionComponent = () => {
   if (!offerResponse) return <React.Fragment></React.Fragment>
   const digitalLocationName = offerResponse.venue.offerer.name
   const locationName = offerResponse.venue.publicName || offerResponse.venue.name
+  const dates = offerResponse.bookableStocks.reduce<Date[]>(
```

Comment:
```suggestion
    const dates = offerResponse.bookableStocks.map((stock) => stock.beginningDatetime).filter(Boolean)
```

Reply:
Ca itère 2 fois sur la liste et Typescript ne reconnait pas bien le type de sortie:
![image](https://user-images.githubusercontent.com/66383742/102476179-15fc7400-405b-11eb-80cf-35d0aea6a8d3.png)
Il faudrait donc re-itérer dessus avec un typeguard :/